docs/src/content/overview-features.md

```diff
@@ -71,17 +70,17 @@ The _separator_ is arbitrary, and is defined by the first character.
 
 ### Examples
 
-Replace `example.org` in all request URLs to with `mitmproxy.org`:
+Map all requests ending with `.jpg` to `https://placedog.net/640/480?random`:
 
-{{< highlight none  >}}
-/example.org/mitmproxy.org
-{{< / highlight >}}
+```
+:.*\.jpg$:https://placedog.net/640/480?random
```

Comment:
Two problems here:
* pattern is parsed incorrectly due to the separator `:` together with the skipped flow filter and the `:` in `https:`. it should instead be `"|.*\.jpg$|https://placedog.net/640/480?random"`
* using a image placeholder service that does not support http2 does not work for websites that do use http2. I debugged this for a while yesterday and then decided to ditch the image placeholder example but use `example.org/image.jpg`. we might want to put a note in the docs (or try to fix the issue, albeit I don't think this problem is a common use case.)

Reply:
other than that, great improvements to the docs :)

Reply:
Good catch on the pattern. 👍 

The HTTP2 issue is a super annoying one, which we'll hopefully be able to fix with sans-io. Let's put in a general note for the addon for now.